assets/sass/modules/_forgot-password.scss

```diff
@@ -35,7 +35,7 @@
     }
   }
 
-  .o-form-input {
```

Comment:
we don't need this for `.o-form-input` anymore?

Reply:
no, we need this style in the external container only which includes also the label